docs/application/vscode-ext/web.md

```diff
@@ -63,87 +63,160 @@ If you do not have a real device, you can run your application in a virtual envi
 
 This section explains how to use VS Code Extension for Tizen Web to develop applications.
 
+### Change Tizen SDK Path
+
+To change a Tizen SDK path, follow these steps:
+
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Change Tizen SDK Path** and press **Enter**.
+3. In the change notification that appears, perform one of the following:
+    - If you do not want to change the path, click **Keep**.
+    - If you want to change the path, click **Change Path**. In the input field of the Command Palette that appears, type the path and press **Enter**.
+
+      > **Note**
+      >
+      > The default path for:
+      >
+      > Window OS: `c:\tizen-studio`
+      >
+      > Linux OS: `/home/username/tizen-studio`
+
+      ![Change Tizen SDK Path](media/tizen_web24.png)
```

Comment:
Global to images. 
For the developer document, it is strongly recommended not to include the author's personal information in the dev doc.
In the new images, I can see "Idh" for the _"user name"_, and guess it is @DohyungLim's personal information.
@DohyungLim do you agree to show your personal information like this in this page?

Reply:
@mijin85cho, thank you for pointing this out.

@DohyungLim, please update the image. Please do not show personal information in images.